src/frontend/src/common/types/location.d.ts

```diff
@@ -0,0 +1,15 @@
+interface GisaidLocation {
+  [index: string]: string | number;
```

Comment:
why did you add this to each type?

Reply:
I did this because I was accessing object properties via string variables, e.g. [this function here](https://github.com/chanzuckerberg/aspen/blob/f144494f4e9b88f46052c3a15fa3f1125160c464/src/frontend/src/views/Upload/components/Metadata/index.tsx#L44-L59)

Reply:
Ended up removing this.